netlify.toml

```diff
@@ -1,3 +1,3 @@
 [build]
 publish = "dist/"
-command = "npm i -g wasm-pack && pnpm run build:wasm && pnpm ci && pnpm run build:js"
+command = "pnpm i --frozen-lockfile && pnpm add -g wasm-pack && pnpm init:biome && pnpm build"
```

Comment:
Put `pnpm i --frozen-lockfile` to the start can make the build script fail fast if the lock file is not updated. So we don't have to waste time on the wasm building process.

`pnpm add -g wasm-pack` should be more cache friendly then `npm i -g wasm-pack` because we use the same package manager. (I don't know how netlify deployment cache works though)